examples/directory/BUILD.bazel

```diff
@@ -67,3 +68,13 @@ swc(
         "minify": False,
     },
 )
+
+bzl_library(
```

Comment:
I should just suppress gazelle here, users don't need to see this in the example

Reply:
fixed in https://github.com/aspect-build/rules_swc/commit/ad8531b0a002f539eafae881338496a94202698e

you can revert the unneeded edits

Reply:
Done.